server/tests/validations/userValidation.test.js

```diff
@@ -0,0 +1,77 @@
+import chai from 'chai';
+import userSchema from '../../validations/userValidation';
+import mockData from '../data/mockData';
+
+const user = mockData[16];
+
+describe('validation test', () => {
+  it('Should validate', () => {

```

Comment:
the tests should be descriptive, all of them are called `should validate` but what is the test validating